database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,10 +14,8 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
```

Comment:
Not entirely sure if maybe the app would benefit from it's own user id? I noticed that @DFurnes has the same structure in [Aurora](https://github.com/DoSomething/aurora/blob/8a3c5319b0b67cf7e8ec8789ce875ea4ab0661e9/database/migrations/2015_01_15_213047_create_users_table.php#L16) and was trying to follow a similar approach, so maybe he can chime in to see if it ended up benefitting to have separate id for app along w/ NS_id...?